subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -60,130 +80,368 @@ typedef struct sicr_key {
 	size_t mac_size;
 } sicr_key;
 
-static sicr_key find_key(uint32_t id)
+typedef struct embedded_key {
+	uint32_t id;
+	uint8_t key_buffer[32];
+	size_t key_buffer_size;
+	psa_key_type_t type;
+	psa_key_bits_t bits;
+} embedded_key;
+
+const embedded_key embedded_keys[] __attribute__((section("_embedded_keys"))) = {
+	{0x4000BB00,
+		{
+#include <public_key_native_MANIFEST_PUBKEY_NRF_TOP_0.bin.inc>
+		},
+	 32,
+	 PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS),
+	 255},
+	{0x4000BB01,
+		{
+#include <public_key_native_MANIFEST_PUBKEY_NRF_TOP_1.bin.inc>
+		},
+	 32,
+	 PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_TWISTED_EDWARDS),
+	 255},
+	{0x4000BB02,
+		{
+#include <public_key_native_MANIFEST_PUBKEY_NRF_TOP_1.bin.inc>
```

Comment:
Should not that be `public_key_native_MANIFEST_PUBKEY_NRF_TOP_2`?

Reply:
will fix